src/signin/SigninCard.js

```diff
@@ -72,10 +72,28 @@ export default class SigninCard extends React.Component {
 					text="Access denied" />
 
 				<LanguagePicker onChange={::this.onLanguageChange} />
-				<button className="outline-link signin-btn" onClick={::this.trySignin}>Sign In</button>
-				<a className="outline-link" target="new" href="https://www.binary.com/user/api_token">Get your API token</a>
+				<button className="outline-link signin-btn" onClick={::this.trySignin}>
+					<FormattedMessage
+						id="sign-in"
+						description="button for sign in"
```

Comment:
Possibly reused for other stuff, not just buttons. I think 'Sign in' is clear enough. May be a description in theory would be 'used to sign in the user into an account' so translators know the context. I'd say leave all empty for now. No 'description' property at all.